src/GolfCourse.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.6;
+
+contract GolfCourse {
```

Comment:
for now I've put all these tests in one contract in one file, but I believe you would like to do one contract/file per example - please confirm